README.md

```diff
@@ -518,7 +525,7 @@ per second at several frequencies with a signal analyzer reference level setting
 
 When an action is run with the above calibration, SCOS will expect the action to have
 a sample_rate, frequency, and reference_level specified in the action config. The values
```

Comment:
This example file also requires preamp_enable and attenuation

Reply:
To resolve this, I opted to adjust the example to remove preamp_enable and attenuation as cal parameters. This makes a smaller example that is a bit easier to parse. (in 2eb839f4eec5d249aa2661b16b5e5a4103382b2f)